snuba/web/rpc/v1/endpoint_trace_item_table.py

```diff
@@ -107,11 +107,18 @@ def _build_query(request: TraceItemTableRequest) -> Query:
 
 
 def _build_snuba_request(request: TraceItemTableRequest) -> SnubaRequest:
+    query_settings = HTTPQuerySettings()
+
+    if request.meta.enable_trace_logs:
+        query_settings.set_clickhouse_settings(
+            {"send_logs_level": "trace", "log_profile_events": 1}
+        )
+
```

Comment:
Refactoring: To implement this functionality across all RPC endpoints, I will create a common method or function that sets up the query settings with trace logs enabled.